lib/commands/alert.js

```diff
@@ -39,10 +39,13 @@ commands.setAlertText = async function (text) {
   }
 };
 
-commands.postAcceptAlert = async function () {
+commands.postAcceptAlert = async function (opts = {}) {
   try {
     let method = 'POST';
-    let endpoint = `/alert/accept`;
+    let endpoint = '/alert/accept';
+    if (opts.buttonLabel) {
+      endpoint += `?name=${encodeURIComponent(opts.buttonLabel)}`;
```

Comment:
Weird. But ok. If another method crops up like that, we should probably refactor `proxyCommand` to handle it.